src/FloatingButton/FloatingButton.less

```diff
@@ -0,0 +1,51 @@
+@import (reference) "../less/utilities";
+
+.FloatingButton--container {
+  position: fixed;
+  .zIndex--9();
+}
+
+.FloatingButton--left {
+  .items--start();
+}
+
+.FloatingButton--center {
+  .items--center();
+  left: 50%;
+  transform: translate(-50%, 0);
```

Comment:
Can we be consistent in our use of positioning? If we're using flexbox positioning for our other positions, can we use flexbox here as well?

Reply:
.items--center(); is consistent with items--end and items--start (align-items: [flex-start,center,flex-end] respectively).

I don't believe the left/transform are possible to achieve with flex (and I'm not using flex for the positioning elsewhere)

Reply:
Ah I see. I was also confused by these but now understand. We're using `items--start/center/end` for the buttons with respect to each other. But need explicit left/right/top/button values for the fixed positioning of the button group on the page. And for the left and right cases, we pass in styles via the `style` prop in the TSX code